data/funnywords.txt

```diff
@@ -0,0 +1,16 @@
+FunnyPrefix1 funny
+FunnyPrefix1 cute
+FunnyPrefix1 adoring
+FunnyPrefix1 cherish
+FunnyPrefix2 little
+FunnyPrefix2 big
+FunnyPrefix2 pink
+FunnyPrefix2 blue
+FunnyCharacter elephant
+FunnyCharacter rhino
+FunnyCharacter mouse
+FunnyCharacter octopus
+FunnySuffix from outer space
+FunnySuffix from the big city
+FunnySuffix from the animal farm
+FunnySuffix from candy heaven
```

Comment:
For even easier parsing and editing - create a directory data/funnywords containing a file for each part:

 - data/funnywords/FunnyPrefix
 - data/funnywords/FunnyCharacter
 - data/funnywords/FunnySuffix

In addition there is no need to distinguish between FunnyPrefix1 and FunnyPrefix2, as mixing those up if anything makes names even funnier: e.g. "blue yellow duck from outer space" or "huge small ant from the big city". Even duplication is harmless: "big big ant from candy heaven", "yellow yellow mouse from the big city" - does not matter.